content/docs/command-reference/fetch.md

```diff
@@ -17,22 +17,17 @@ positional arguments:
 
 ## Description
 
-Downloads DVC-tracked files from remote storage into the cache of the project
-(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
-This makes them available for linking (or copying) into the workspace (refer to
-[`dvc config cache.type`](/doc/command-reference/config#cache)).
+Downloads tracked data from remote storage into the <abbr>cache</abbr> of the
+project (without placing it in the <abbr>workspace</abbr> like `dvc pull`). This
+makes them available for linking (or copying) into the workspace (refer to
+`dvc config cache.type`).
```

Comment:
where will it link it?

Reply:
Not sure I understand the Q. DVC will link them into the workspace as mentioned (that text didn't change in this PR BTW).

Reply:
I guess instead of referring to `dvc config` we can use `dvc checkout` here... Updated.